Examples/Algorithms/Geant4/src/EventAction.hpp

```diff
@@ -53,7 +53,7 @@ class EventAction final : public G4UserEventAction {
   ///
   /// This only contains valid data after the end-of-event action has been
   /// executed.
-  const std::vector<Acts::RecordedMaterialTrack>& materialTracks() const;
+  std::vector<Acts::RecordedMaterialTrack> materialTracks();
```

Comment:
I think we can not make a `State&` visitor pattern here, but since this is G4UserEventAction, I guess we can just leave it as such.